gov/governance/_GET_proposal.gno

```diff
@@ -1,113 +1,37 @@
 package governance
```

Comment:
change the file name to `getter.gno` and combine this with `_GET_vote.gno`. Add unit tests.